src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.cs

```diff
@@ -70,6 +72,7 @@ internal CurlHandler()
                 throw new HttpRequestException(SR.net_http_client_execution_error);
             }
             SetCurlMultiOptions();
+            PollFunction(_multiHandle);
```

Comment:
I see you added a call to PollFunction in AddEasyHandle, but this call is still here.